net-ftp/uftpd/files/0001-remove-license.patch

```diff
@@ -0,0 +1,11 @@
+--- uftpd-2.5/Makefile.am	2018-06-06 19:32:37.000000000 +0200
++++ uftpd-2.5/Makefile.am.1	2018-06-11 09:07:10.227975263 +0200
+@@ -1,6 +1,6 @@
+ SUBDIRS            = src man
+-doc_DATA           = README.md LICENSE
+-EXTRA_DIST         = README.md LICENSE ChangeLog.md
++doc_DATA           = README.md
++EXTRA_DIST         = README.md ChangeLog.md
```

Comment:
Actually, modifying `EXTRA_DIST` incorrect. Not that it matters for Gentoo, so I won't block you on this.